daemons/based/based_callbacks.c

```diff
@@ -45,8 +45,8 @@ static int cib_process_command(xmlNode *request,
                                cib__op_fn_t op_function, xmlNode **reply,
                                xmlNode **cib_diff, bool privileged);
 
-static gboolean cib_common_callback(qb_ipcs_connection_t *c, void *data,
-                                    size_t size, gboolean privileged);
+static int32_t cib_common_callback(qb_ipcs_connection_t *c, void *data,
```

Comment:
Until/unless we do want to return something useful, I'd prefer to return `void`. That makes it clearer that we're ignoring the result (or rather, that there is no result). However, I'm not gonna hold up this PR over that, as it's just preference.